content/en/docs/contributing/policy.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Feature Policy"
+linkTitle: "Feature Policy"
+weight: 70
+type: "docs"
+---
+
+We are open to feature requests and PRs implementing those. If you plan on contributing a new feature into cert-manager we recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
+
+# Features we will not allow
+
+Certain features have ben requested before but are not able to being implemented properly or not able to being implemented without breaking the security model.
+
+Below is a list of a few of these:
+
+## Vendoring Kubernetes related APIs outside of the `k8s.io/` namespace
+
+Vendoring Kubernetes related APIs like OpenShift specific classes is not recommend as it vendors the `k8s.io/apimachinery` package itself could potentially conflict with cert-manager's instance. It could also cause a conflict with different Kubernetes client versions being used.
```

Comment:
I would rephrase slightly to describe the generic problem with specific examples.

Vendoring project APIs that also vendor `k8s.io/apimachinery`, such as OpenShift, Contour, or Velero, is not recommend because the kubernetes dependency is likely to conflict with cert-manager's instance.